src/components/Modals/NewMessageModal.jsx

```diff
@@ -107,12 +107,7 @@ const NewMessageModal = ({ showModal, setShowModal, oldMessage = '', toField = '
         }, 2000);
```

Comment:
@xscottxbrownx looks like this `setTimeout()` is causing that.

Reply:
Looks like the setTimeout was part of Development too. I've only touched on code related to the message lists here in this PR.

Think we could open up a new PR to handle this, we do have a problem with the modal closing regardless of error. We could get that resolved as well by how we handle the two set functions inside the `finally` block, whether we move them out or change the functions, etc. That'll be good.

https://github.com/codeforpdx/PASS/assets/14917816/fc7cf06d-25a4-4ab9-ad73-9b846273ebc7